Makefile

```diff
@@ -188,6 +188,15 @@ test/addons/.buildstamp: config.gypi \
 # TODO(bnoordhuis) Force rebuild after gyp update.
 build-addons: $(NODE_EXE) test/addons/.buildstamp
 
+ifeq ($(OSTYPE),$(filter $(OSTYPE),darwin aix))
+  XARGS = xargs
+else
+  XARGS = xargs -r
+endif
+clear-stalled:
+	ps awwwx | grep Release/node | grep -v grep | cat
```

Comment:
Does the third `w` in the ps options add something? Docs and anecdotal usage indicate more than 2 doesn't make a difference.